packages/v2/buidler.config.js

```diff
@@ -0,0 +1,45 @@
+const { usePlugin } = require('@nomiclabs/buidler/config')
+
+usePlugin("@nomiclabs/buidler-ganache")
+usePlugin('@nomiclabs/buidler-truffle5')
+
+const ACCOUNTS = (process.env.ETH_KEYS ? process.env.ETH_KEYS.split(',') : [])
```

Comment:
At some point we should start using sth like [buidler-local-networks-config-plugin](https://github.com/facuspagnuolo/buidler-local-networks-config-plugin)